### ISSUE TITLE
[Bug]: Erro

### ISSUE THREAD

**Issue by dimitri-weidriti** [BlafKing/sd-civitai-browser-plus#167]
### Describe the bug.

The search works fine; it only gives an error when I try to search for Lore or Checkpoint by clicking on the Civai icon

Watch the video for a better understanding

https://github.com/BlafKing/sd-civitai-browser-plus/assets/153913984/8ea86c2c-1edd-417f-96ab-f97a0570a981

### Steps to reproduce the problem.

1. Go to .... 
2. Press ....
3. ...

### Expected behavior

Have him search for the lore and the checkpoint when I click on the icon, as it used to work before.

### System info

* Extension version: [281cdc04] 
* OS: win 11
* SD-WebUI version: Version: 1.7.0
* Python: Python 3.10.10

### Console logs

```Shell
Traceback (most recent call last):
  File "C:\Users\bruno\stable-diffusion-webui\venv\lib\site-packages\gradio\routes.py", line 488, in run_predict
    output = await app.get_blocks().process_api(
  File "C:\Users\bruno\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 1431, in process_api
    result = await self.call_function(
  File "C:\Users\bruno\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 1103, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "C:\Users\bruno\stable-diffusion-webui\venv\lib\site-packages\anyio\to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "C:\Users\bruno\stable-diffusion-webui\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "C:\Users\bruno\stable-diffusion-webui\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "C:\Users\bruno\stable-diffusion-webui\venv\lib\site-packages\gradio\utils.py", line 707, in wrapper
    response = f(*args, **kwargs)
  File "C:\Users\bruno\stable-diffusion-webui\extensions\sd-civitai-browser-plus\scripts\civitai_file_manage.py", line 369, in model_from_sent
    gl.json_data = _api.api_to_data(content_type, "Newest", "AllTime", "Model name", None, None, None, tile_count, f"civitai.com/models/{modelID}")
  File "C:\Users\bruno\stable-diffusion-webui\extensions\sd-civitai-browser-plus\scripts\civitai_api.py", line 175, in api_to_data
    model_number = match.group(1)
AttributeError: 'NoneType' object has no attribute 'group'
```

**Comment by rowild** [comment]
I can confirm the same problem.

**Comment by BlafKing** [comment]
Thanks for reporting!

The issue ended up being caused by selecting some models that do not exist on the CivitAI servers, I have now included some better description and logging to let the user know what the cause of the issue was.

So this issue is now fixed in ([v3.2.5](https://github.com/BlafKing/sd-civitai-browser-plus/commit/c3129a7fd3d4cc63378027725e8fc67de15fdedb))

Feel free to send another msg here if you still experience any issues! :)
